cnames_active.js

```diff
@@ -882,6 +882,7 @@ var cnames_active = {
   "dragonman225": "dragonman225.github.io",
   "draw": "skillzzjesse.github.io/draw",
   "dream": "joeleeofficial.github.io/dream",
+  "dreamlo": "jmheartley.github.io/dreamlo.js"
   "dreamcord": "dreamcordjs.github.io/dreamcord",
```

Comment:
```suggestion
  "dreamcord": "dreamcordjs.github.io/dreamcord",
  "dreamlo": "jmheartley.github.io/dreamlo.js",
```
As the CI error notes, your entry needs to be in alphabetical order, and is missing a comma at the end of the line